### DISCUSSION TITLE
The sftp protocol login function is not synchronized when Active Directory changes the user password or disables the user.

### DISCUSSION THREAD

**Discussion by jojo642** [drakkan/sftpgo#1642] [General]
The sftp protocol login function is not synchronized when Active Directory changes the user password or disables the user.
The web functions are synchronized

**Comment by drakkan** [comment]
Hello, thanks for using SFTPGo.
Please avoid asking the same question over and over again. This is the third time. If someone wants to take time to understand your problem and help you maybe you will get an answer.

More info

https://github.com/drakkan/sftpgo?tab=readme-ov-file#sponsors
https://github.com/drakkan/sftpgo?tab=readme-ov-file#support-policy

Good luck with your project

**Reply by pwnall1337** [reply]
"Hi, I see you found a bug in my software, have a nice day". 

So basically because LDAP plugin stores the credentials and doesn't update when the password changes ldap server side, the ldap plugin or using ldap with sftpgo is effectively useless.

Thanks
